arch/arm/mach-imx/mxs/Kconfig

```
@@ -78,15 +78,7 @@ config TARGET_PWSH7
 	select BOARD_EARLY_INIT_F
 
 config TARGET_PWA7200
-	bool "Support PW-A7200"
-	select BOARD_EARLY_INIT_F
-
-config TARGET_PWG4200
-	bool "Support PW-G4200"
-	select BOARD_EARLY_INIT_F
-
-config TARGET_PWG5300
-	bool "Support PW-G5300"
+	bool "Support PW-A7200 series"
```

Describing all compatible models will help other users.

```suggestion
	bool "Support PW-A7200 series"
	help
	  Compatible to: G4200, G5200, G5300, A7200, A7300, A7400, A9100, A9200, A9300
```